runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenConfig.java

```diff
@@ -22,19 +22,39 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.usernamesyncer;
+package net.runelite.client.plugins.loginscreen;
 
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup(
-	keyName = "rememberusername",
-	name = "Username Syncer",
-	description = "Syncs the username that is currently remembered between computers"
+	keyName = "loginscreen",
+	name = "Login Screen",
+	description = "Provides various enhancements for login screen"
 )
-public interface UsernameSyncerConfig extends Config
+public interface LoginScreenConfig extends Config
 {
+	@ConfigItem(
+		keyName = "syncusername",
+		name = "Sync username",
+		description = "Syncs the username that is currently remembered between computers"
+	)
+	default boolean syncUsername()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "pasteenabled",
+		name = "Ctrl-V paste",
+		description = "Enables Ctrl + V key binding that will set username or password on login screen from clipboard."
+	)
```

Comment:
The wording here is a bit strung-out, maybe something simpler like `Enables Ctrl + V pasting on the login screen` ?

Reply:
Well it is description, I think it should mention what it actually does, it is not real paste, it replaces entire field with the clipboard contents.

Reply:
I guess that's true since it replaces any other text in the field, but I still think the wording is slightly unnatural.